doc/source/options.rst

```diff
@@ -392,6 +392,9 @@ display.width              80           Width of the display in characters.
                                         IPython qtconsole, or IDLE do not run in a
                                         terminal and hence it is not possible
                                         to correctly detect the width.
+display.html.table_schema  True         Whether to publish a Table Schema
+                                        representation for frontends that
```

Comment:
hmm, didn't realize we have a ``html.`` namespace here. and of course we have ``display.notebook.html_repr`` which should prob also be under ``html`` namespace. I am liking this ``display.html.*`` namespace.